tests/e2e/verfication_post_day1_test.go

```diff
@@ -336,5 +336,12 @@ var _ = Describe("TF Test", func() {
 				}
 			})
 		})
+		Context("Author:zhsun-Critical-OCP-63950 @zhsun", func() {
+			It("Verify imdsv2 value is set post cluster creation", ci.Day1Post, ci.Critical, func() {
+				if profile.Ec2MetadataHttpTokens != "" {
```

Comment:
@sunzhaohua2 If it is empty, then it should have the default value of "optional" right?

Reply:
yes, will add a check